chain-core/src/state/account.rs

```diff
@@ -51,3 +57,91 @@ impl Account {
         hash256::<Blake2s>(&self.address)
     }
 }
+
+/// attributes in account-related transactions
+#[derive(Debug, Default, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode)]
+pub struct AccountOpAttributes {
+    pub chain_hex_id: [u8; 1],
```

Comment:
Why an array of size `1` and not just `u8`?

Reply:
oops, that was a temporary laziness to get encode/decode derive